test/uk/gov/hmrc/debttransformationstub/utils/ifsrulesmasterspreadsheet/InterestForecastingRulesGeneratorSpec.scala

```diff
@@ -99,7 +99,7 @@ final class InterestForecastingRulesGeneratorSpec extends AnyFreeSpec {
             "6010\t1554\tN\t\tY\tN/A",
             "SA SSTTP Debts",
             "Main Trans\tSub Trans\tInterest bearing\tInterest key\tInterest only Debt\tCharge Ref",
-            "4910\t1005\tY\t\tN\tN/A",
+            "4910\t1005\tY\t\tN\tVRN",
```

Comment:
Whats this change? Still going to approve but I'm confused. Is it left in from playing around?